bin/objectbox_model_generator/lib/src/code_chunks.dart

```diff
@@ -0,0 +1,46 @@
+import "package:objectbox/src/modelinfo/index.dart";
+
+class CodeChunks {
+  static String modelInfoLoader(String allModelsJsonFilename) => """
```

Comment:
The parameter appears unused. Is it kept for backwards compatibility?

Reply:
removed